rabbitizer/rabbitizer.pyi

```diff
@@ -166,7 +188,20 @@ class Instruction:
     def getAccessType(self) -> Enum: ...
     def doesUnsignedMemoryAccess(self) -> bool: ...
 
-    def disassemble(self, immOverride: str|None=None, extraLJust: int=0) -> str: ...
+    def disassemble(self, immOverride: str|None=None, extraLJust: int=0) -> str:
+        """Returns a string that can be assembled back to the instruction raw word.
+        `immOverride` can be a string that replaces the immediate in the disassembly,
+        or the jump target, if any. If the instruction has neither, it is ignored.
+        Examples:
+        >>> Instruction(0x3C0A0042).disassemble()
+        'lui         $t2, 0x42'
+        >>> Instruction(0x3C0A0042).disassemble("hex_answer")
+        'lui         $t2, hex_answer'
+        >>> Instruction(0x0C000042).disassemble()
+        'jal         func_80000108'
+        >>> Instruction(0x0C000042).disassemble("my_target")
+        'jal         my_target'
```

Comment:
Do you know if vscode will properly highlight those examples if you write them this way? Like, it doesn't need to be enclosed on some kind of block (like \`\`\` \`\`\`)

Reply:
works on my machine tm
![image](https://github.com/Decompollaborate/rabbitizer/assets/5306619/93f34304-03c9-4afb-97be-d49d672f7f46)

I think this is somewhat loosely standard for examples